lib/chewy/search/response.rb

```diff
@@ -33,6 +39,17 @@ def max_score
         @max_score ||= hits_root['max_score']
       end
 
+      # Response `errors` field. Returns `nil` if there is no error.
+      # @return [Array, nil]
+      def errors
+        errors = Array(@body['errors'])
+        # there's another case when failures could be present in some shards, in that case es returns them
+        # in @body[_shards][failures] array
+        errors += Array(@body['_shards']['failures']) if @body['_shards']
+        # presence will return nil if array is empty
```

Comment:
Is it more intuitive signature though :)

Reply:
The `Array` method was handling the `nil` and single element check, so used it directly.
```ruby
x = [0] + Array(1) + Array([2, 3, 4]) + Array(nil)
=> [0, 1, 2, 3, 4]
```
Feels ok to me, do you have any suggestions?